Cargo.toml

```diff
@@ -44,6 +44,7 @@ hacl-star = { version = "=0.0.11", optional = true }
 rand = { version = "^0.5", optional = true }
 ring = { version = "^0.13.0", optional = true }
 x25519-dalek = { version = "^0.3", optional = true, default-features = false, features = ["std", "u64_backend"] }
+secp256k1 = "0.11.3"
```

Comment:
You might consider the pure Rust `k256` crate from as an alternative here:

https://github.com/RustCrypto/elliptic-curves/tree/master/k256